App/views.py

```diff
@@ -6,7 +6,7 @@
 def main(request):
     if(request.method=='POST'):
         usr = request.POST['username']  #Here, we get the username fetched from html page
-        usr = usr.replace("&","%26")
+        usr = usr.replace("&","")
```

Comment:
That will just break the URL.

Reply:
Sir so should I consider input containing "&" as invalid , not deal with trying correct the handle ? Please tell . Currently , if I enter a username ey bd&808 , it converts to bd808 and displays the result.

Reply:
bd&808 is a different (valid) username though so that would prevent that person from using your tool. The percent-encoding approach wasn't wrong, you just did it at the wrong place (also, no point in limiting it to just `&`). See my earlier comment - you should do escaping in the template, and use whatever tools the templating language provides.

Reply:
Sir, in django templating, escaping is done by default on. Now, I have used 'safe' filter in my html file , so the username will not be auto-escaped. Am I going right now ?